site/templates/application/footer/social-media-buttons.jet

```diff
@@ -0,0 +1,22 @@
+{{block footerSocialMediaButtons()}}
+  {{social_link_instagram := config("social_link_instagram")}}
+  {{social_link_youtube := config("social_link_youtube")}}
+  {{social_link_facebook := config("social_link_facebook")}}
+  {{social_link_twitter := config("social_link_twitter")}}
+  {{social_link_letterboxd := config("social_link_letterboxd")}}
+  {{if social_link_instagram != "" || social_link_youtube != "" || social_link_facebook != "" || social_link_twitter != "" || social_link_letterboxd != ""}}
+    <div class="footer-social-media-icons">
+    {{if social_link_instagram != ""}} {{ yield socialMediaIcon(link=social_link_instagram,social="instagram") }} {{end}}
+    {{if social_link_youtube != ""}} {{ yield socialMediaIcon(link=social_link_youtube,social="youtube") }} {{end}}
+    {{if social_link_facebook != ""}} {{ yield socialMediaIcon(link=social_link_facebook,social="facebook") }} {{end}}
+    {{if social_link_twitter != ""}} {{ yield socialMediaIcon(link=social_link_twitter,social="twitter") }} {{end}}
+    {{if social_link_letterboxd != ""}} {{ yield socialMediaIcon(link=social_link_letterboxd,social="letterboxd") }} {{end}}
+    </div>
+  {{end}}
+{{end}}
+
+{{block socialMediaIcon(link,social)}}
+      <a target="_blank" title="{{social}}" href="{{link}}" class="footer-social-media-icon {{social}}">
```

Comment:
Some issues on this `<a>`.  Have a look at [this guy](https://github.com/indiereign/shift72-web/blob/526cc56e9deb36be3de2ceab21d3d96458b886b5/src/ui/modal/share-modal.component.js#L66) as an example.

- The class should be `s72-btn`.  Your `footer-social-media-icon` and variable `"social"` classes have no purpose so they can be removed.
- Class should be the first attribute for readability.
- The title attribute needs to run through the `i18n` function (we already have translations for these social media link titles).
- You should move the conditional inside this `socialMediaIcon` block so it doesn't have to be repeated each time in the `footerSocialMediaButtons` block.
- I think a more consistent name for this block would be `footerSocialMediaButton`.

Reply:
Also, the indents are borked here :P